grapheneapi/graphenewsrpc.py

```diff
@@ -35,10 +38,26 @@ class GrapheneWebsocketRPC(object):
     api_id = {}
 
     def __init__(self, url, user="", password=""):
-        self.url = url
+        if isinstance(url, list):
+            self.url_list = url
+            if not isinstance(self.url_iterator, collections.Iterable):
+                self.url_iterator = cycle(self.url_list)
+            self.url = next(self.url_iterator)
+            for num in range(len(self.url_list)):
```

Comment:
`net()` gives is the next url in the list, why do you use a for loop here?.

Side remark: instead of `range(len())` you can use `enumerate()`